ideascube/db_router.py

```diff
@@ -1,30 +1,44 @@
+
+_databaseToUse = {
+    ('search', 'search') : 'transient'
+}
```

Comment:
So you're adding a mapping here, in addition to the one on the model?

Why not simply get the model from the `app_label` and `model_name`?